### ISSUE TITLE
TinyBert prediction-layer distillation loss : Why not KLDiv ?

### ISSUE THREAD

**Issue by astariul** [huawei-noah/Pretrained-Language-Model#12]
For the prediction-layer distillation loss, you used the **soft cross-entropy loss**.

However, other distill architecture (Distil-Bert, Bert-PKD) used **KLDiv loss** for the prediction layer.

---

**What is the reason behind this choice ?**
Did you try KLDiv and it gave worse results ? Is it empirical choice ?

**Comment by nlpBeginner** [comment]
> For the prediction-layer distillation loss, you used the **soft cross-entropy loss**.
> 
> However, other distill architecture (Distil-Bert, Bert-PKD) used **KLDiv loss** for the prediction layer.
> 
> **What is the reason behind this choice ?**
> Did you try KLDiv and it gave worse results ? Is it empirical choice ?

Actually, we have not tested the KDDiv loss and will consider it in our future experiment.
